exporter/syslogexporter/README.md

```diff
@@ -0,0 +1,63 @@
+- `tls` - configuration for TLS/mTLS
+  - `insecure` (default = `false`) whether to enable client transport security, by default, TLS is enabled.
+  - `cert_file` - Path to the TLS cert to use for TLS required connections. Should only be used if `insecure` is set to `false`.
+  - `key_file` - Path to the TLS key to use for TLS required connections. Should only be used if `insecure` is set to `false`.
+  - `ca_file` - Path to the CA cert. For a client this verifies the server certificate. For a server this verifies client certificates. If empty uses system root CA. Should only be used if `insecure` is set to `false`.
+  - `insecure_skip_verify` -  (default = `false`) whether to skip verifying the certificate or not.
+  - `min_version` (default = `1.2`) Minimum acceptable TLS version
+  - `max_version` (default = `""` handled by [crypto/tls][cryptoTLS] - currently TLS 1.3) Maximum acceptable TLS version.
+  - `reload_interval` - Specifies the duration after which the certificate will be reloaded. If not set, it will never be reloaded.
+- `retry_on_failure`
+  - `enabled` (default = `true`)
+  - `initial_interval` (default = `5s`): Time to wait after the first failure before retrying; ignored if `enabled` is `false`
+  - `max_interval` (default = 30s): Is the upper bound on backoff; ignored if `enabled` is `false`
+  - `max_elapsed_time` (default = `120s`): Is the maximum amount of time spent trying to send a batch; ignored if `enabled` is `false`
+- `sending_queue`
+  - `enabled` (default = `false`)
+  - `num_consumers` (default = `10`): Number of consumers that dequeue batches; ignored if `enabled` is `false`
+  - `queue_size` (default = `5000`): Maximum number of batches kept in memory before data; ignored if `enabled` is `false`;
+  User should calculate this as `num_seconds * requests_per_second` where:
+    - `num_seconds` is the number of seconds to buffer in case of a backend outage
+    - `requests_per_second` is the average number of requests per seconds.
+  - `storage` (default = `none`): When set, enables persistence and uses the component specified as a storage extension for the [persistent queue][persistent_queue]
+- `timeout` (default = 5s) Time to wait per individual attempt to send data to a backend
+
+Please see [example configurations](./examples/).
+
+[RFC5424]: https://www.rfc-editor.org/rfc/rfc5424
+[RFC3164]: https://www.rfc-editor.org/rfc/rfc3164
+[syslog_parser]: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/pkg/stanza/docs/operators/syslog_parser.md
+[syslog_receiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/syslogreceiver
+[filelog_receiver]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/filelogreceiver
+[cryptoTLS]: https://github.com/golang/go/blob/518889b35cb07f3e71963f2ccfc0f96ee26a51ce/src/crypto/tls/common.go#L706-L709
+[development]: https://github.com/open-telemetry/opentelemetry-collector#development
```

Comment:
This line is probably not needed anymore, as the link is being autogenerated together with the status table.
```suggestion
```

Reply:
@kkujawa-sumo, please take note of this. It can be done in a follow up PR.